sticky_notes.py

```diff
@@ -81,3 +81,37 @@ def list_stickynotes(self, event):
                 self.delay_task(index,
                                 self.sender(format_note(index, note), event))
             return first_note
+
+    @cmd
+    def del_note(self, event):
```

Comment:
@mrshu not a place here but how about we implemented some kind of secondary command system? Maybe even in `brutal-ng` itself, that class would have predefined top-level command and then every `@cmd` method of such class would be secondary command, for example: `sn delete` would be the same as `del_stickynote` or `sn list` would behave the same as `list_stickynotes`, etc.